lib/utils.pm

```diff
@@ -1095,6 +1095,11 @@ sub set_hostname {
     save_screenshot;
 
     if (systemctl('is-active NetworkManager', ignore_failure => 1) == 0) {
+        if (is_s390x || is_spvm || is_pvm_hmc) {
```

Comment:
`is_pvm = is_spvm || is_pvm_hmc` IMO:)

Reply:
btw, I am not sure if `ipmi` can be considered as well

Reply:
hhmm, yes, I think I'll just try and avoid having to check for all these things.